src/Lean/Elab/MutualInductive.lean

```diff
@@ -983,13 +980,14 @@ private def elabInductiveViews (vars : Array Expr) (elabs : Array InductiveElabS
   withExporting (isExporting := !isPrivateName view0.declName) do
     let res ← mkInductiveDecl vars elabs
     mkAuxConstructions (elabs.map (·.view.declName))
+    -- Call `enableRealizationsForConst` now as later constructions may need `noConfusion`
+    for e in elabs do
+      enableRealizationsForConst e.view.declName
```

Comment:
This means that a realizable constant construction will not be able to use the sizeOf, below or injectivity theorems (unless these are also turned into realizable constants), right? (That’s fine, just something one should be be aware of).